testutil/integration/doc.go

```diff
@@ -0,0 +1,2 @@
+// Integration contains the integration test suite used for SDK modules.
```

Comment:
```suggestion
// Integration contains the integration test setup used for SDK modules.
```
`suite` might not be the proper word

Reply:
I agree.